### DISCUSSION TITLE
Install Instructions (video)

### DISCUSSION THREAD

**Discussion by jimafisk** [muety/wakapi#590] [Show and tell]
Just found this project today and thought it was pretty rad! Awesome work @muety thanks so much for sharing it with everyone!

I made a video of installing it locally and connecting to vscodium: https://youtu.be/iEb71_kaluc?si=upKaodn5Vj4m6hb7

Hope it helps new folks get a quick look at the project. Thanks!

**Comment by muety** [comment]
Great to hear that, thank you so much! 🙏 Glad you like Wakapi!
